src/main/java/uk/ac/ebi/eva/evaseqcol/controller/seqcol/SeqColComparisonController.java

```diff
@@ -23,6 +24,7 @@
 
 @RestController
 @RequestMapping("/comparison")
+@Tag(name = "Comparison endpoints")
```

Comment:
Technically, these are also under the seqcol specification so they should be called Seqcol as well.
I would actually prefer if they are in the same section.

Reply:
This looks good, my nitpicks:
- I wonder if we can at least get the two `comparison` endpoints to sit next to each other in the same section?
- There's a typo in the tag name, should be Se**qc**ol!

Reply:
Done ! 
Thanks April

Reply:
Now it looks like this:
![Screenshot from 2024-01-29 17-24-46](https://github.com/EBIvariation/eva-seqcol/assets/82417779/4cfd360c-22bd-4bd0-8c38-94d66e7844cc)